docs/blog/newsletter-april-2019.rst

```diff
@@ -0,0 +1,114 @@
+.. post:: April 3, 2019
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – April 2019
+######################################
+
+Hello folks! Beth and the newsletter team here, welcoming you to the April edition of the newsletter.
+
+Spring is here and that means conference time is speeding towards us! Portland (19-21 May) is *very nearly* sold out; I'm excited for those of you who've got tickets, the lineup looks fab. There's tons of things to tell you about the conference and I'm not even going to try to cover them all - check out the `blog post </conf/portland/2019/news/events-activities/>`_ for details on childcare, diversity tickets, shirts, and much more. 
+
+In Europe, ticket sales and CFPs are open for both Prague and the brand new Vilnius conference. The `Vilnius CFP </conf/vilnius/2019/cfp/>`_ closes **22 April**, and the `Prague CFP </conf/prague/2019/cfp/>`_ closes **30 April**. We can't wait to hear your ideas! 
+
+So that's the news this month. Read on for stories from Slack, plenty of job listings, and tons of meetups around the globe!
+
+-------------------------------
+Page length and user preference
+-------------------------------
+
+This month, the WTD community discussed whether users prefer shorter or longer pages in documentation. Both options have upsides and downsides, but most documentarians seemed to feel that longer pages work better for users; and some reported their user feedback has been strongly in favor of longer pages. Here are some of the issues folks discussed:
+
```

Comment:
Should the semi-colon in the second sentence be a comma? "...for users, and some..."

Reply:
I've gone semi-colon mad in this post, haven't I? Fixed.